src/platform/user/profile/vet360/components/PhoneField/PhoneEditModal.jsx

```diff
@@ -77,6 +77,7 @@ class PhoneEditModal extends React.Component {
           this.props.data.phoneNumber
         }`,
         extension: this.props.data.extension || '',
+        isTextPermitted: this.props.data.isTextPermitted || false,
```

Comment:
VA Profile would give us `null` values here, which could lead to a form validation error because the schema specifies that `isTextPermitted` must be a boolean.

Reply:
Oof, frustrating. Do you think this would make sense to be in that function for converting to clean data before submit?

Reply:
I kinda prefer doing the cleanup up front, but I don't see one way being particularly better than the other ¯\_(ツ)_/¯

Reply:
Ah, I misunderstood part of this. I think your method makes more sense 👍